plugins/fullstory.client.js

```diff
@@ -0,0 +1,3 @@
+import * as FullStory from '@fullstory/browser'
+
+FullStory.init({ orgId: process.env.FULLSTORY_ORG_ID })
```

Comment:
LGTM, probably only need to call this init when the `process.env.CONTEXT === 'production'`, looks like the tests are failing because this `init` is expecting an `orgId`

Reply:
the `CONTEXT` env var is set by Netlify btw, which is why I recommend using a check like so:

```js
if (process.env.CONTEXT === 'production') {
  FullStory.init(// ...
}
```

Reply:
@matthamil great call out. I do need to add that check.